src/main/java/io/cdap/delta/transformation/MaskTransformation.java

```diff
@@ -49,30 +66,32 @@ private void parseDirective(TransformationContext context) {
       throw new IllegalArgumentException("Directive command line is null.");
     }
     String[] splits = commandLine.split(" ");
-    if (splits.length != 4) {
-      throw new IllegalArgumentException("Directive should have 4 arguments. Usage: mask column_name from_position " +
-                                           "to_position.");
+    if (splits.length != 5) {
+      throw new IllegalArgumentException("Directive is missing some parts or containing more parts," +
+                                           "Expected: mask column_name direction mask_char n, " +
+                                           "given directive: " + commandLine);
     }
     if (!NAME.equals(splits[0])) {
       throw new IllegalArgumentException("Directive is not a mask transformation. Usage: mask column_name " +
-                                           "from_position to_position.");
+                                           "direction mask_char n.");
     }
 
     srcColumn = splits[1];
-    try {
-      fromPos = Integer.parseInt(splits[2]);
-    } catch (NumberFormatException e) {
-      throw new IllegalArgumentException("from_position is not an integer.", e);
+    direction = splits[2];
+    if (!(direction.equalsIgnoreCase(RIGHT_DIRECTION) || direction.equalsIgnoreCase(LEFT_DIRECTION))) {
```

Comment:
if we are making directive name lower case, then we should also make direction lower case to be consistent.

Reply:
+1